.github/repository-settings.md

```diff
@@ -36,7 +36,7 @@ Same settings as above for `main`, except:
 
   (So that opentelemetrybot can create release branches)
 
-### `dependabot/**/**` and `opentelemetrybot/*` and `gradlew-update-*`
+### `dependabot/**/**`, `renovate/**/**`, `opentelemetrybot/*`, and `gradlew-update-*`
```

Comment:
Should we remove `dependabot/**/**`?

Reply:
I'll send a separate PR later once we've onboarded successfully to remove dependabot from everywhere